swagger/v1/swagger.yaml

```diff
@@ -2,7 +2,8 @@
 openapi: 3.0.1
 info:
   title: LAA Legal Framework
-  description: This service provides read-only lookup data relating to applications for Civil legal aid. [It is primarily used by the Apply for civil legal aid service]
+  description: This service provides read-only lookup data relating to applications
```

Comment:
**nit** hmm, what happened to the description - automated change?!

Reply:
This was an automated change (by running NOCOVERAGE=true rake rswag) - have rebased on main to check but the description still changes.

Reply:
It was added by @naseberry in https://github.com/ministryofjustice/legal-framework-api/pull/1004/files
Weird that it's getting removed again! 😕